content/index.html

```diff
@@ -2,6 +2,46 @@
 title: Home
 ---
 
+# Up next
+
+<%
+    (upcoming_talks(@items)).sort_by do |item|
+        item.attributes[:date].to_datetime
+    end.last(1) do |item|
```

Comment:
You don't need to pass `1` to `last`.

You also cannot pass a block to `last`. So `last` returns the `item` here and you can simply do `item = (upcoming...`, I think.